frontend/src/components/AddUpdate/AddShoppingUpdate.tsx

```diff
@@ -0,0 +1,85 @@
+            <label className="label-input-update">Number of Souvenirs: <input className="number-field"
+                                                                               type="number"
+                                                                               placeholder="Add Amount of additional bought small Souvenirs"
+                                                                               min="0" step="1"
+                                                                               value={amountOfSouvenirItems}
+                                                                               onChange={event => setAmountOfSouvenirItems(Number(event.target.value))}/>
+            </label>
+            <label className="label-input-update">Custom Shopping Item: <input className="text-field"
+                                                                                type="customShoppingItem"
+                                                                                placeholder="Add an additional Custom Shopping Item"
+                                                                                value={customShoppingItem}
+                                                                                onChange={event => setCustomShoppingItem(event.target.value)}/>
+            </label>
+            <label className="label-input-update"> Amount of Custom Shopping Item: <input className="number-field"
+                                                                                           type="number"
+                                                                                           placeholder="Add Amount of additional bought Custom Shopping Items"
+                                                                                           min="1" step="1"
+                                                                                           value={amountOfCustomShoppingItem}
+                                                                                           onChange={event => setAmountOfCustomShoppingItem(Number(event.target.value))}/>
+            </label>
+            <label className="label-input-update"> Emissions of Custom Shopping Item: <input className="number-field"
+                                                                                              type="number"
+                                                                                              placeholder="Add Emissions of your additional Custom Shopping Item"
+                                                                                              min="0" step="0.1"
+                                                                                              value={customShoppingItemEmission}
+                                                                                              onChange={event => setCustomShoppingItemEmission(Number(event.target.value))}/>
+            </label>
```

Comment:
Unless Custom Shopping Item and some abstraction would shorten the code and might make it more readable.